transcripts/package.json

```diff
@@ -10,9 +10,9 @@
     "dependencies": {
         "@types/node": "^10.12.18",
         "@types/restify": "^7.2.1",
-        "botbuilder": "^4.2.1",
-        "botbuilder-ai": "^4.2.1",
-        "botbuilder-dialogs": "^4.2.1"
+        "botbuilder": "4.1.6",
```

Comment:
I don't understand this set of changes.

Reply:
By having a different dependency version that was unpinned, `lerna bootstrap --hoist` was fetching this version and placing it at the root of the repository.

All `4.1.6` versions (e.g. the libraries and test bot) were being correctly symlinked, while this package.json was telling lerna to fetch `^4.2.1` from npm.

Without this unpinned version, the `SkypeMentionNormalizeMiddleware` class was not successfully building because `botbuilder-core` is not a dependency of `botbuilder-core`.

[See my previous comment.](https://github.com/microsoft/botbuilder-js/pull/1437#discussion_r349251015)

Reply:
> _Without this unpinned version, the SkypeMentionNormalizeMiddleware class was not successfully building because botbuilder-core is not a dependency of botbuilder-core._

To clarify, by setting the version here to 4.1.6, the dependencies in the transcripts/package.json were being symlinked.

This meant that there were no `botbuilder-core` dependencies at the root for the SkypeMentionNormalizeMiddleware class to resolve against, which caused the `botbuilder-core` library to not build.